checker/check_result.go

```diff
@@ -262,3 +264,22 @@ func LogFindings(findings []finding.Finding, dl DetailLogger) {
 		}
 	}
 }
+
+// IsExempted verifies if a given check in the results is exempted in maintainers annotation.
+func (check *CheckResult) IsExempted(maintainersAnnotation ma.MaintainersAnnotation) (bool, []string) {
+	// If check has a maximum score, then there it doesn't make sense anymore to reason the check
+	// This may happen if the check score was once low but then the problem was fixed on Scorecard side
+	// or on the maintainers side
+	if check.Score == MaxResultScore {
+		return false, nil
```

Comment:
If we remove this check it allows repo maintainers to self-report false negatives in Scorecard (i.e. reasons the score shouldn't be 10/10 even if it is). Not required, but just a thought.

Reply:
Good point. We can do that, but my guess is that the annotation will be currently used to reason low scores, not high scores. So, I would avoid reasoning that for now, see if the feature is being used or not, and then add this later on if the users need it. Do you agree?

Reply:
Yeah, sounds good!